packages/network-controller/tests/provider-api-tests/block-hash-in-response.ts

```diff
@@ -274,4 +279,707 @@ export function testsForRpcMethodsThatCheckForBlockHashInResponse(
       });
     });
   }
+
+  describe.each([
```

Comment:
This file (which is used to test two RPC methods) didn't have any tests around handling of responses. I originally wrote them for block-param.ts and no-block-param.ts and then copied them over.